### ISSUE TITLE
Kurento-room-server does not have a mainClass defined in pom.xml

### ISSUE THREAD

**Issue by tylkomat** [Kurento/bugtracker#13]
I use openjdk-7 and receive the following error:
```
Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.3.2:java (default-cli) on project kurento-room-server: The parameters 'mainClass' for goal org.codehaus.mojo:exec-maven-plugin:1.3.2:java are missing or invalid
```

**Comment by igracia** [comment]
It seems to me that you are executing `mvn exec:java`, not compiling. The kurento-room-server is not supposed to be used like that, as it has to be embedded in a Spring-Boot application. Please have a look at [this project](https://github.com/Kurento/kurento-room/tree/master/kurento-room-demo) for an executable kurento-room-server.

I think there is a mistake in the documentation on how to run this. We'll have that corrected.

@rvlad Could you please have a look at the documentation for the kurento-room project? It seems that there are some misleading instructions there, namely this part
```
$ git clone git@github.com:Kurento/kurento-room.git
$ cd kurento-room
$ git checkout 6.1.0
$ mvn compile exec:java
```
That should be executed in the proper folder I guess.

**Comment by rvlad** [comment]
@igracia you're right, there is an errata, the Maven command should be executed from the `kurento-room-demo` folder.
```bash
$ git clone git@github.com:Kurento/kurento-room.git
$ cd kurento-room-demo
$ git checkout 6.1.0
$ mvn compile exec:java
```

**Comment by igracia** [comment]
@rvlad Thanks!
@tylkomat We've updated the README.md, but the patch is now in our CI. It will go to github soon. We'll close the issue once the patch is merged. Thanks for reporting this!

**Comment by rvlad** [comment]
Fixed in commit 5c59c6ee919cce1f8955b9c54884856fb5186854
